modules/nlu/src/backend/confusion-engine.ts

```diff
@@ -26,6 +27,10 @@ export default class ConfusionEngine extends ScopedEngine {
     await super.init()
   }
 
+  public getConfusionStatus() {
+    return this.confusionStatus
+  }
```

Comment:
Unless you make additional computation, you can simply set the variable public

Reply:
or if you want to make sure the outside world doesn't write it. you cout write a getter

```js
get confustionStatus(){
  return this.confustionStatus
}
```